templates/typesetting/typesetting_review_assignment.html

```diff
@@ -16,48 +16,58 @@
-                      <th>Created</th>
-                      <th>Due</th>
-                      <th>Accepted</th>
-                      <th>Completed</th>
-                      <th>Status</th>
-                  </tr>
-                  <tr>
-                      <td>{{ assignment.pk }}</td>
-                      <td>{{ assignment.manager.full_name }}</td>
-                      <td>{{ assignment.assigned|date }}</td>
-                      <td>{{ assignment.due|date }}</td>
-                      <td>{{ assignment.accepted|date }}</td>
-                      <td>{% if assignment.completed %}
-                          {{ assignment.completed|date }}{% else %}
-                          {{ assignment.completed|bool_fa }}{% endif %}</td>
-                      <td>{{ assignment.status|capfirst }}</td>
-                  </tr>
-                  {% if assignment.accepted %}
-                      <tr>
-                          <th colspan="7">Note from Typesetter</th>
-                      </tr>
-                      <tr>
-                          <td colspan="7">{{ assignment.typesetter_note|safe }}</td>
-                      </tr>
-                  {% endif %}
-              </table>
-          </div>
-          </div>
+        <div class="title-area">
+          <h2>Typesetting worked on
+            by {{ assignment.typesetter.full_name }}</h2>
+          <a href="{% url 'typesetting_article' article.pk %}" class="button"><
+            Back</a>
+        </div>
+        <div class="content">
+          <table class="table small">
+            <tr>
+              <th>ID</th>
+              <th>Manager</th>
+              <th>Created</th>
+              <th>Due</th>
+              <th>Accepted</th>
+              <th>Completed</th>
+              <th>Status</th>
+            </tr>
+            <tr>
+              <td>{{ assignment.pk }}</td>
+              <td>{{ assignment.manager.full_name }}</td>
+              <td>{{ assignment.assigned|date }}</td>
+              <td>{{ assignment.due|date }}</td>
+              <td>{{ assignment.accepted|date }}</td>
+              <td>{% if assignment.completed %}
+                {{ assignment.completed|date }}{% else %}
+                {{ assignment.completed|bool_fa }}{% endif %}</td>
+              <td>{{ assignment.status|capfirst }}</td>
+            </tr>
+            {% if assignment.accepted %}
+              <tr>
+                <th colspan="7">Note from Typesetter</th>
+              </tr>
+              <tr>
+                <td colspan="7">
+                  {{ assignment.typesetter_note|safe|default:"No typesetter note recorded." }}
+                </td>
+              </tr>
+            {% endif %}
+          </table>
+        </div>
+        <div class="title-area">
+          <h2>Typesetting Task</h2>
+        </div>
+        <div class="content">
+          {{ assignment.task|safe|default:"No typesetting task was recorded." }}
+        </div>
```

Comment:
Intuitively, I would expect this information to come before the typesetter's note, which will in many cases be a reply to the assignment text.

I also would expect it to look similar to the typesetter's note, but the way you've done it here, the assignment has an h2 and title styling, but the note is in a table. Can we make them similar?

Last, the files selected are going to be a key piece of information going forward, since that is communication between project editor and typesetter. Can you add the list of files selected and the list of galleys requested to be changed above the assignment text?